### PR TITLE
Minor fixes for Python 4.0 release

### PR DESCRIPTION
**Pull request by maria-robobug**

### CODE REVIEW

File: modules/project-docs/pages/sdk-release-notes.adoc [hunk 1]
```diff
@@ -9,11 +9,8 @@
 [abstract]
 {description}
 
-include::hello-world:page$start-using-sdk.adoc[tag=prep]
+Couchbase Python SDK 4.x is built upon Couchbase++, and SDK 3.x is built upon LCB (libcouchbase), but both conform to the xref:project-docs:compatibility.adoc#api-version[SDK API 3.x].
```
Comment (maria-robobug) [inline modules/project-docs/pages/sdk-release-notes.adoc]: Not sure what is going on here, but for some reason the xref doesn't work unless I add an empty line after this.

See current issue in prod: https://docs.couchbase.com/python-sdk/current/project-docs/sdk-release-notes.html

File: modules/hello-world/examples/hello_world_t.py [hunk 2]
```diff
@@ -19,14 +19,17 @@
...
     # We strongly reccomend this for production use.
     # cert_path=cert_path
 )
 
 # Get a reference to our cluster
 # NOTE: For TLS/SSL connection use 'couchbases://<your-ip-address>' instead
 cluster = Cluster('couchbase://localhost', ClusterOptions(auth))
+
+# Wait until the cluster is ready for use.
+cluster.wait_until_ready(timedelta(seconds=5))
```
Comment (maria-robobug) [inline modules/hello-world/examples/hello_world_t.py:32]: Adding `cluster.wait_until_ready` which is new in SDK 4.0

File: modules/project-docs/pages/sdk-release-notes.adoc (hunk 1 above)
Comment (RichardSmedley) [reply modules/project-docs/pages/sdk-release-notes.adoc]: We also need a macro in attributes for Couchbase++
It's showing here as Couchbase.

Comment (RichardSmedley) [reply modules/project-docs/pages/sdk-release-notes.adoc]: (We have one for C++ for just that reason)

File: modules/project-docs/pages/sdk-release-notes.adoc [hunk 3]
```diff
@@ -36,7 +33,7 @@ For example, to install version 3.2.6:
 $ python3 -m pip install git+https://github.com/couchbase/couchbase-python-client@3.2.6
 ----
 
-For more detailed installation instructions, see the xref:project-pages:sdk-full-installation.adoc[full installation guide].
+For more detailed installation instructions, see the xref:project-docs:sdk-full-installation.adoc[full installation guide].
```
Comment (RichardSmedley) [inline modules/project-docs/pages/sdk-release-notes.adoc:35]: Good spot - thought I'd got to all of these!

File: modules/project-docs/pages/sdk-release-notes.adoc (hunk 1 above)
Comment (maria-robobug) [reply modules/project-docs/pages/sdk-release-notes.adoc]: Ah, I had forgotten that, hmm I can't seem to find where `cpp(C++)` is defined.
I could add a `cbpp` variable to take care of that in docs-site? Then we should be able to reference it anywhere in the docs.

Like the `sql++` wording here: https://github.com/couchbase/docs-site/blob/master/antora-playbook.yml#L160

Comment (maria-robobug) [reply modules/project-docs/pages/sdk-release-notes.adoc]: Just tried locally and works nicely.

Comment (RichardSmedley) [reply modules/project-docs/pages/sdk-release-notes.adoc]: :partying_face:

**Review by RichardSmedley — APPROVED** [review]
:+1:

**Pull request merged**